tool/travis/sass-spec-ref.sh

```diff
@@ -7,7 +7,7 @@
 # run. If we're running specs for a pull request which refers to a sass-spec
 # pull request, we'll run against the latter rather than sass-spec master.
 
-default=master
+default=feature.use
```

Comment:
Should this still be master on this branch?

Reply:
Yes! Good catch!

Reply:
Then, next question I just though of... Why was travis green when running the feature.use spec when it should be failing?